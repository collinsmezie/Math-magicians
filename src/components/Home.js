import React from 'react';
import './Home.css';

function Home() {
  return (

    <div className="homepage">
      <h2>Welcome to our page</h2>
      <p>
        &quot; Math magicians &quot; is a website for all fans of mathematics.
        It is a Single Page App (SPA) that allows users to make simple calculations
        and read a random math-related quote.
      </p>

    </div>
  );
}

export default Home;
