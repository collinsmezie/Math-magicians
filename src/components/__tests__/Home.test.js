import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Home from '../Home';

describe('Home page of the App', () => {
  test('Builds the snapshot of the Home component safely', () => {
    const rend = renderer.create(<Home />);
    expect(rend.toJSON()).toMatchSnapshot();
  });
  test('Render Home component', () => {
    render(<Home />);
  });
});
