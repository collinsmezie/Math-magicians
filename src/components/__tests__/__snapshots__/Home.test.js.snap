// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page of the App Builds the snapshot of the Home component safely 1`] = `
<div
  className="homepage"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    " Math magicians " is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
  </p>
</div>
`;
